dev-aws/customer-billing/topics.tf

```diff
@@ -0,0 +1,330 @@
+resource "kafka_topic" "finance-invoice-model" {
+  name               = "finance-invoice-model"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    "compression.type" = "zstd"
+    "retention.bytes" = "805306368000"
```

Comment:
The retention bytes looks like a big value (750 GB / partition)
I think you could get it lower.
Would be also nice to put a comment with what the human readable value is.
If you don't need infinite retention, you should put "cleanup.policy"   = "delete".
You may consider overwriting the "retention.ms", which is 7 days by default.
Take a look at this one: https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/dev-aws/kafka-shared/otel.tf#L5C13-L17

Reply:
That makes all sense. We converted the configuration from the old cluster without thinking. Will update it and select smaller more reasonable values. We removed the cleanup.policy because delete is the default based on https://docs.confluent.io/platform/current/installation/configuration/topic-configs.html#cleanup-policy. But we can add it back.

Reply:
We have updated it and added some comments too. The values are still not perfect but reduced it by a factor of 100 for now.

Reply:
We decided to mirror the retention.ms we currently have in dev, which uses the default for all but 3 topics.

We will put more thought into future topics we will add